### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by balderdash**
Remove the deprecation notice, add `--s3Prefix` flag to the README.

### CODE REVIEW

**Review by jghazarian — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -17,7 +13,7 @@ cache.
 
 *   Launch `bazels3cache` like this (by default it listens on port 7777):
 
-        bazels3cache --bucket=MY_S3_BUCKET
```
Comment (msb336) [inline README.md:20]: is `MY_S3_BUCKET` user specific, or should we all be pointing to the same aws bucket?

Comment (msb336) [reply README.md:20]: @balderdash ^

Comment (balderdash) [reply README.md:20]: We'll all use the same bucket. This is a public fork of bazels3cache so I'm leaving the placeholder arguments here. I'll put the specific arguments you'll want to use in the audere mediapipe README.

Comment (msb336) [reply README.md:20]: ahh got it. Sorry I though I was looking at the `mediapipe` repo

**Review by msb336 — APPROVED** [review]

**Pull request merged**